pkg/stub/handler.go

```diff
@@ -304,6 +305,43 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+//TODO leaving this in for now to promote discussion of this which started in planning and enhancement proprosal;
+// can delete if need be as PR review process completes 
+func (h *Handler) Disconnected() bool {
```

Comment:
I agree @adambkaplan which is why the call to this is commented out.

Between anyone on @openshift/openshift-team-developer-experience and @bparees I was hoping to brainstorm a bit on the specifics of how we would do this.  If not here in the PR, we could use this as a starting point for discussion in scrum or maybe the team arch meeting.  But if folks have immediate thoughts, please do chime in.

Otherwise I can remove it from this PR and add it to a separate one after we are done here with IPv6 to 

Thoughts?